app/models/card.rb

```diff
@@ -15,39 +19,24 @@ class Card < ActiveRecord::Base
 
   mount_uploader :image, CardImageUploader
 
-  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
-  scope :repeating, -> { where('quality < ?', 4).order('RANDOM()') }
-
-  def check_translation(user_translation)
-    distance = Levenshtein.distance(full_downcase(translated_text),
-                                    full_downcase(user_translation))
-
-    sm_hash = SuperMemo.algorithm(interval, repeat, efactor, attempt, distance, 1)
-
-    if distance <= 1
-      sm_hash.merge!({ review_date: Time.now + interval.to_i.days, attempt: 1 })
-      update(sm_hash)
-      { state: true, distance: distance }
-    else
-      sm_hash.merge!({ attempt: [attempt + 1, 5].min })
-      update(sm_hash)
-      { state: false, distance: distance }
-    end
-  end
+  scope :pending, -> {
+    where("review_date <= ?",
+    Time.zone.now).order("RANDOM()")
+  }
+  scope :repeating, -> { where("quality < ?", 4).order("RANDOM()") }
 
   def self.pending_cards_notification
     users = User.where.not(email: nil)
     users.each do |user|
-      if user.cards.pending.any?
-        CardsMailer.pending_cards_notification(user.email).deliver
-      end
+      return unless user.cards.pending.any?
```

Comment:
Non-local exit from iterator, without return value. `next`, `break`, `Array#find`, `Array#any?`, etc. is preferred.